cleanup_utils/integration_cleanup.py

```diff
@@ -89,12 +94,21 @@ def run_integration_cleanup(dry_run=False):
         'region_name': os.environ['OPENSTACK_REGION'],
     }
     os_cleanup = OpenStackCleanupInstance(
-        age_limit=DEFAULT_AGE_LIMIT,
+        age_limit=DEFAULT_CUTOFF_TIME,
         openstack_settings=openstack_settings,
         dry_run=dry_run
     )
     os_cleanup.run_cleanup()
 
+    # Run MySQL cleanup
+    mysql_cleanup = MySqlCleanupInstance(
+        age_limit=DEFAULT_AGE_LIMIT,
+        url=os.environ['DEFAULT_INSTANCE_MYSQL_URL'],
+        domain=os.environ['DEFAULT_INSTANCE_BASE_DOMAIN'],
+        dry_run=args.dry_run
```

Comment:
@jamestait, there is no variable `args` in this method - it was removed in @giovannicimolin's recent PR which you had to rebase your changes on. So how did this even work? 🤔

```suggestion
        dry_run=dry_run
```

Reply:
@lgp171188 As the args definition happens outside a function, it becomes a global variable. Nevertheless, I've created `run_integration_cleanup` method so we can run the cleanup from python shell if needed, and this `args.dry_run` would throw an error, so it must be changed to the parameter `dry_run` as suggested.

Reply:
Great catch, thanks! Fixed.